src/game/scenes/party.js

```diff
@@ -2,64 +2,101 @@ import Phaser from 'phaser';
+    this.createImage(0, 0.5, 0.55, 'enna');
+    this.createImage(1, 0.9, 0.85, 'crowd5');
+    this.createImage(1, 0, 0.4, 'switch');
+    this.createImage(1, 0.2, 0.6, 'reimu');
+    this.createImage(1, 1, 0.7, 'tv');
+    this.createImage(2, 0.8, 0.55, 'nina');
+    this.createImage(2, 0.5, 0.1, 'banner', (image) => {
+      // Banner = Message Board
+      image
+        .setInteractive({ cursor: `url(${Cursor2}), auto`, pixelPerfect: true })
+        .on('pointerdown', () => {
+          this.game.vue.dialog = true;
+          this.game.vue.openProject = 'banner';
+        });
+    });
+    this.createImage(3, 0.5, 0.5, 'room');
+    this.createImage(3, 1, 0.5, 'gifts');
+    this.createImage(3, 0.2, 0.45, 'artworks', (image) => {
+      // Artwork = Drawing Board
+      image
+        .setInteractive({ cursor: `url(${Cursor2}), auto`, pixelPerfect: true })
+        .on('pointerdown', () => {
+          this.game.vue.dialog = true;
+          this.game.vue.openProject = 'artworks';
+        });
+    });
 
-    // Cake = Birthday Relay
-    this.images.cake.obj
-      .setInteractive({ cursor: `url(${Cursor2}), auto`, pixelPerfect: true })
-      .on('pointerdown', () => {
-        this.game.vue.dialog = true;
-        this.game.vue.openProject = 'cake';
+    this.input.on('pointermove', (pointer) => {
+      Object.entries(this.containers).forEach(([level, container]) => {
+        const newX = centerX - ((pointer.x - centerX) * (0.017 * level));
+        const newY = centerY - ((pointer.y - centerY) * (0.008 * level));
+        container.setPosition(newX, newY);
       });
+    });
+  }
 
-    // Artwork = Drawing Board
-    this.images.artworks.obj
-      .setInteractive({ cursor: `url(${Cursor2}), auto`, pixelPerfect: true })
-      .on('pointerdown', () => {
-        this.game.vue.dialog = true;
-        this.game.vue.openProject = 'artworks';
-      });
+  createContainers(quantity, maxLevel) {
+    const { width, height } = this.sys.game.canvas;
+    const centerX = width / 2;
+    const centerY = height / 2;
+    let level = maxLevel;
 
-    // Banner = Message Board
-    this.images.banner.obj
-      .setInteractive({ cursor: `url(${Cursor2}), auto`, pixelPerfect: true })
-      .on('pointerdown', () => {
-        this.game.vue.dialog = true;
-        this.game.vue.openProject = 'banner';
-      });
+    for (let index = quantity - 1; index >= 0; index -= 1) {
+      const container = this.add.container(centerX, centerY);
+      this.containers[`${level}`] = container;
+      level -= 1;
+    }
+  }
 
-    this.input.on('pointermove', (pointer) => {
-      Object.values(this.images).forEach((image) => {
-        const newX = centerX + ((pointer.x - centerX) * (0.017 * image.level));
-        const newY = centerY + ((pointer.y - centerY) * (0.008 * image.level)) - 20;
-        image.obj.setPosition(newX, newY);
-      });
-    });
+  // x and y are values from 0 to 1
+  createImage(level, x, y, texture, cb) {
```

Comment:
this is open for discussion, whether we use pixel or by percentage relative to the container